velox/functions/prestosql/tests/ArraySortTest.cpp

```diff
@@ -229,7 +229,7 @@ class ArraySortTest : public FunctionBaseTest,
         test<StringView>();
         break;
       case TypeKind::MAP:
-        test<TestMapType>();
+        // Ignore test for MAP for it is not an orderable type.
```

Comment:
TestMapType and all the code using it is now dead code. Let's remove it.

Reply:
Done